packages/theme/src/cli/commands/theme/pull.ts

```diff
@@ -70,6 +70,8 @@ If no theme is specified, then you're prompted to select the theme to pull from
       only: flags.only,
       ignore: flags.ignore,
       force: flags.force,
+      verbose: flags.verbose,
+      noColor: flags['no-color'],
```

Comment:
We use this pattern for flags with complex names instead of camelCase, which we use in the code

https://github.com/Shopify/cli/blob/a1604b2ece020d138125a9c99e41083404756c6d/packages/cli-kit/src/public/node/cli.ts#L194